runtime/missingmember_test.go

```diff
@@ -4408,9 +4408,9 @@ pub contract ExampleNFT {
 
         pub fun deposit(token: @NFT)
 
-        pub fun getIDs(): [UInt64]
+        view pub fun getIDs(): [UInt64]
 
-        pub fun idExists(id: UInt64): Bool
+        view pub fun idExists(id: UInt64): Bool
```

Comment:
Why does this function have to be `view` now?

Reply:
It was called from within a `getIds`, but can be non-view now.